ui-new/css/main.scss

```diff
@@ -86,6 +101,379 @@ footer {
   padding-right: 2vh;
 }
 
+.mainv .actions .button-yes {
+  margin-left: 2.5vw;
+  margin-right: 2.4vw;
+  border-radius: 4vh;
+  padding: 2vh 3vh;
+}
+
+.mainv .actions .button-no {
+  margin-left: 2.4vw;
+  margin-right: 2.5vw;
+  border-radius: 4vh;
+  padding: 2vh 3vh;
+}
+
+.cash-out-color .mainv .actions .button-yes {
+  border: 2px solid $purple-grey;
+  background: $purple-grey;
+  color: $white;
+  @include museo-bold
+  @include Description-2-bold
+}
+
+.cash-out-color .mainv .actions .button-no {
+  border: 2px solid $purple-grey;
+  color: $purple-grey;
+  @include museo-bold
+  @include Description-2-bold
+}
+
+.cash-out-color .mainv .actions .button-yes:active,
+.cash-out-color .mainv .actions .button-no:active {
+  border: 2px solid $white;
+  background: $white;
+  color: $purple-grey;
+  @include museo-bold
+  @include Description-2-bold
+}
+
+.cash-in-color .mainv .actions .button-yes {
+  border: 2px solid $sapphire;
+  background: $sapphire;
+  color: $white;
+  @include museo-bold
+  @include Description-2-bold
+}
+
+.cash-in-color .mainv .actions .button-no {
+  border: 2px solid $sapphire;
+  color: $sapphire;
+  @include museo-bold
+  @include Description-2-bold
+}
+
+.cash-in-color .mainv .actions .button-yes:active,
+.cash-in-color .mainv .actions .button-no:active {
+  border: 2px solid $white;
+  background: $white;
+  color: $sapphire;
+  @include museo-bold
+  @include Description-2-bold
+}
+
+.deposit_network_down_state .mainv .header {
```

Comment:
This part of the css is one of the things I was going to talk to you about. I'm assuming every screen has similar definitions and they are used to correctly place the header icon and h3/h2 on the same position across the screens.

I did the following for similar screens
- Set a header with fixed height
- Set a main with adjustable height
- footer that can be pushed off but with a set height

With this I can create several of these screens on a couple of hours and create a couple of standards based on the different header heights. 

https://github.com/lamassu/lamassu-machine/pull/247/files#diff-fad9caab4b96a0c206ae0c71ef18c0b8R524

Reply:
Yeah, your approach is way better!